From 4ee1421a176eca4d9df4bba640b91071594eeb61 Mon Sep 17 00:00:00 2001
From: Jun Nie <jun.nie@linaro.org>
Date: Thu, 8 Aug 2019 12:20:02 +0800
Subject: [PATCH 3/3] iMX8M: Del FIP entry in FIT image

This patch dels FIP entry in the mkimage script. Because we want to
flash FIP to SD card separately so that FIP can be updated without
touching ATF.

Signed-off-by: Jun Nie <jun.nie@linaro.org>
---
 iMX8M/mkimage_fit_atf.sh | 12 +-----------
 1 file changed, 1 insertion(+), 11 deletions(-)

diff --git a/iMX8M/mkimage_fit_atf.sh b/iMX8M/mkimage_fit_atf.sh
index b35b4fc..1dda520 100755
--- a/iMX8M/mkimage_fit_atf.sh
+++ b/iMX8M/mkimage_fit_atf.sh
@@ -49,8 +49,6 @@ do
 	ls -lct $dtname | awk '{print $5}' >&2
 done
 
-FIP="fip.bin"
-
 cat << __HEADER_EOF
 /dts-v1/;
 
@@ -67,14 +65,6 @@ cat << __HEADER_EOF
 			load = <$ATF_LOAD_ADDR>;
 			entry = <$ATF_LOAD_ADDR>;
 		};
-		fip@1 {
-			description = "ARM Trusted Firmware FIP";
-			data = /incbin/("$FIP");
-			type = "firmware";
-			arch = "arm64";
-			compression = "none";
-			load = <0x40310000>;
-		};
 __HEADER_EOF
 
 
@@ -93,7 +83,7 @@ cat << __CONF_SECTION_EOF
 		config@$cnt {
 			description = "$(basename $dtname .dtb)";
 			firmware = "uboot@1";
-			loadables = "atf@1", "fip@1";
+			loadables = "atf@1";
 		};
 __CONF_SECTION_EOF
 else
-- 
2.7.4

